app/packages/files/file.ts

```diff
@@ -60,6 +60,10 @@ export default abstract class File {
         return 'read';
     }
 
+    get currentUserCanDelete() {
+        return this.fileModel.target.get('modelName') !== 'registration';
```

Comment:
It looks like readonly users shouldn't be able to delete files either

Reply:
Not going to worry about this now. I think @brianjgeiger should have this fixed in his PR once the permission stuff is up.